melodies_monet/util/env_sub.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+def env_sub(file_str):
+    """Substitute envirnoment variables prefixed by $
+    in file location string.
+
+    Parameters
+    -------
+    -------
+    file_str (str): file string
+
+    Returns
+    -------
+    file_str (str): modified file string
+    """
+    for subpath in file_str.split("/"):
```

Comment:
To be safe / for OS compat. https://stackoverflow.com/q/3167154
```suggestion
    from pathlib import Path

    for subpath in Path(file_str).parts:
```